cider/exec/nextgen/jitlib/llvmjit/LLVMJITModule.cpp

```diff
@@ -81,9 +89,47 @@ void LLVMJITModule::finish() {
   llvm::outs() << *module_;
 
   LLVMJITEngineBuilder builder(*this);
+
+#ifndef WITH_JITLIB_DEBUG
```

Comment:
Q: Why by-default off for ```OptimizeIR```?

Reply:
In Debug model, we expect to get the whole IR information. so by-default off, we will do some optimization in different optimize level